front/app/components/ProjectAndFolderCards/components/EmptyContainer.tsx

```diff
@@ -90,10 +94,12 @@ const EmptyContainer = () => {
       />
       <EmptyMessage>
         <EmptyMessageTitle>
-          <FormattedMessage {...messages.noProjectYet} />
+          {/* <FormattedMessage {...messages.noProjectYet} /> */}
```

Comment:
Is this left on purpose?

Reply:
Nope, forgot about those! Thanks for spotting